pdns/misc.cc

```diff
@@ -597,7 +599,7 @@ void shuffle(vector<DNSZoneRecord>& rrs)
       break;
 
   if(second-first > 1)
-    random_shuffle(first,second);
+    shuffle(first, second, std::default_random_engine(seed));
```

Comment:
It appears `default_random_engine` is implementation defined, probably including the constructor signature. Should we really use this?

I'm also concerned about providing a static seed of zero to an implementation defined random engine.

https://www.cplusplus.com/reference/random/default_random_engine/ suggests that it could be a [linear_congruential_engine](https://www.cplusplus.com/reference/random/linear_congruential_engine/linear_congruential_engine/)
 which has a default seed value (of 1u)...